### ISSUE TITLE
Guideline Candidate: Informed Consent

### ISSUE THREAD

**Issue by pawn002** [pawn002/applied-ethics-in-design#8]
I’m pretty confident at this point that people must be given the choice more often than not.

**Comment by coreygearhart** [comment]
For most cases I would agree....but isn't informed consent tied to expectations of privacy? For instance, if a group (maybe working within an intranet) has no expectation of privacy, could informed consent simply be repetitive?

- https://uxdesign.cc/thoughts-on-consent-ethics-in-tracking-user-behaviour-for-ux-research-e0c9d018d935 (Just including this as a pretty good article on informed consent in UX)

**Comment by pawn002** [comment]
To be clear, @coreygearhart are you stating that repetition is something we should avoid as it can be safely assumed in some contexts?

If so, could you dissect my personal view that no one reads End User License Agreements and as such (because of the predominance of this behavior), one shouldn't expect someone to be bound by it.

**Comment by coreygearhart** [comment]
Yes, I think if consent can be assumed in some environments it is not needed. And some methods when it cannot be assumed are broken. 

Funny you should mention those agreements, I was just listening to a [podcast](https://www.npr.org/2020/03/04/812264543/episode-976-terms-of-service) recently about those. The end supposition was that if a company needs to put their shady practices into an End User License Agreement, maybe those practices should be banned. 
I agree that the End User License System is broken, if users do not read them they should not be bound by them.
